R/splat-estimate.R

```diff
@@ -13,7 +13,8 @@
 #' \code{\link{splatEstOutlier}}, \code{\link{splatEstBCV}},
 #' \code{\link{splatEstDropout}}
 #'
-#' @return SplatParams object containing the estimated parameters.
+#' @return SplatParams or splatPopParams object containing the estimated 
```

Comment:
Think it's fine to just say SplatParams here because SplatPopParams is a subclass.